### ISSUE TITLE
Teller new currently fails in master branch

### ISSUE THREAD

**Issue by rayterrill** [tellerops/teller#200]
## Expected Behavior
teller new should successfully build a .teller.yml file.

## Current Behavior
Fails with the following error:
```
FATA[0004] could not create configuration                error="template: t:276: function \"stage\" not defined"
```

## Steps to Reproduce

1. clone master branch
2. teller new, pick any secret provider
3. get error
4.

## Specifications
  - Version: `$ teller version`
  - Platform:

**Comment by rayterrill** [comment]
Looks like this potentially broke on this PR: https://github.com/tellerops/teller/commit/358cd8393a72c1661e6876ddc39e52c6475b30b3

**Comment by jondot** [comment]
thanks, resolved part of Teller 2.0
